### ISSUE TITLE
Dark mode for search bar

### ISSUE THREAD

**Issue by zachmullen** [girder/girder_web_components#170]
Right now the search bar doesn't look good in a light context:

![Screen Shot 2019-10-07 at 11 40 37 AM](https://user-images.githubusercontent.com/555959/66326855-c1f2ed00-e8f7-11e9-9de2-9b531dc76ece.png)

Note that the icon is invisible, and the contrast is poor.

@mgrauer in this case the light background is kind of forced on us due to the fact that the DANDI logo really only works on a light background due to the text. This is not a problem, just documenting why I care, and I assume other downstreams will care also, since light backgrounds are very common.

So this is another DANDI feature request for @subdavis

**Comment by subdavis** [comment]
Light search results in a `v-menu` under a dark search bar will look weird, so that would need to be dark too.  Before taking on this work, I'd like to have a plan for how to deal with light/dark in every component.  I'm fine with implementing one-at-a-time though.

**Comment by zachmullen** [comment]
Agreed about an overall plan. I think this component is unique in GWC in that it's essentially a single text field input, other than the results. For DANDI, I'd be fine with the search results still being light, it's more an issue of the text field contrast, and the icon.

**Comment by mgrauer** [comment]
For icons that need a light background, can we use a dark theme and just add a small light background div around the icon?
